docs/add-code-flow.md

```diff
@@ -100,4 +96,8 @@ Within the function, a new `Adder` is created with the configured `Blockstore` a
 
   - **[`adder.PinRoot()`](https://github.com/ipfs/go-ipfs/blob/v0.4.18/core/coreunix/add.go#L171)** - *Pin all files under the `MFS` **root***
 
-    The whole process ends with `PinRoot` recursively pinning all the files under the `MFS` **root**
\ No newline at end of file
+    The whole process ends with `PinRoot` recursively pinning all the files under the `MFS` **root**
+
+# Also see
+1. https://github.com/ipfs/go-ipfs/#development
+2. [Concept document about files](https://github.com/ipfs/docs/issues/133)
```

Comment:
We can probably move this to the introduction (before actually starting to talk about MFS/UnixFS).